hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/request/DefaultEntityRequestBuilder.java

```diff
@@ -154,5 +188,7 @@ private TimeRangeArgument timeRange(Map<String, Object> arguments) {
     EdgeSetGroupRequest outgoingEdgeRequests;
     boolean includeInactive;
     boolean fetchTotal;
+    Optional<ContextualRequest> labelRequest;
+    AttributeRequest labelsAttributeRequest;
```

Comment:
We should have a local LabelsRequest POJO which should encapsulate both, label request and label attribute request.

Reply:
done